tools/model/templates.go

```diff
@@ -0,0 +1,60 @@
+package model
+
+import (
+	"html/template"
+
+	"github.com/gobuffalo/flect"
+)
+
+var modelTemplate string = `package models
+
+import (
+	{{- range $i := .Imports }}
+	"{{$i}}"
+	{{- end }}
+)
+
+// {{ properize .Name }} model struct
+type {{ properize .Name }} struct {
+	{{- range $attr := .Attrs }}
+	{{ pascalize $attr.Name }}	{{$attr.GoType }} ` + "`" + `json:"{{ underscore $attr.Name }}" db:"{{ underscore $attr.Name }}"` + "`" + `
+	{{- end }}
+}
+
+// {{ pluralize .Name }} array model struct of {{ properize .Name }}
+type {{ pluralize .Name }} []{{ properize .Name }}
+
+// String converts the struct into a string value
+func ({{ .Char }} {{ properize .Name }}) String() string {
+	js, _ := json.Marshal({{ .Char }})
```

Comment:
@dmuriel I think we should handle error here. (This is a common issue generated models have)

Reply:
I just add a "log.Fatal(err)" on the template.